### ISSUE TITLE
Cannot install Pillow 8.2.0 on Termux

### ISSUE THREAD

**Issue by MangoMan0** [python-pillow/Pillow#7165]
### What did you do?
I'm Student

### What did you expect to happen?
Get successful Install Pillow v8.2.0

### What actually happened?
Gives Error 
https://nekobin.com/megetidofu

### What are your OS, Python and Pillow versions?

* OS: Termux 
* Python: Latest (3.11.3)
* Pillow: 8.2.0

```python
pip install pillow==8.2.0
```

**Comment by bigcat88** [comment]
https://pillow.readthedocs.io/en/stable/installation.html#python-support

Use older Python or a fresher Pillow. Or follow instructions to install Pillow from source.

**Comment by nulano** [comment]
```
      ld.lld: error: /system/lib/libdl.so is incompatible with aarch64linux
      ld.lld: error: /system/lib/libc.so is incompatible with aarch64linux
      ld.lld: error: /system/lib/libdl.so is incompatible with aarch64linux
```
Looks like it is a source install, but there is something wrong with the compiler. Doesn't really look like an issue with Pillow.

**Comment by MangoMan0** [comment]
> 

Thank you I'll try to use older version of python 🌸

**Comment by radarhere** [comment]
To be clear - Pillow 8.2.0 was released before Python 3.11 was released. So we do not officially support Pillow 8.2.0 on Python 3.11. Instead, it would be a better idea to install a version of Pillow that was released after Python 3.11 - Pillow 9.3.0 or later.
